src/main/java/Doug.java

```diff
@@ -0,0 +1,180 @@
+                + "                                     .#(*******,   %,   */      (@%&@&        *.#. ,/, .#/\n"
+                + "                                     .#,     (.     #(          #@&(@@&      *,  /*    ./*#%\n"
+                + "                                     .#,  ./.       .*.(.      .&@@/%@@%    *,    **      (.#/\n"
+                + "                                     .(,  ,/.        .*  ,/    *&@@//@@@(  /.      *,     (  *#\n"
+                + "                                     .#,     */       ./   .(. /@@@#,&@@@//         /.    (   .&.\n"
+                + "                                     .#/,      ./      ,*     ,%@@@%.#@@@%.         .(   ./    .&.\n"
+                + "                                     .#*.,.    .,.(     /.    .#@@@&**@@@@(          **  *,     /(\n"
+                + "                                     .%*.,     .,  ./   .*    .%@@@@#.&@@@@,         (  /     .#.\n";
+
+        System.out.println(face);
+
+
+        // Welcome statement
+        System.out.println(dashedLine + "Howdy partner! My name is Doug Dimmadome.\n"
+                + "Now what can I do for ya?\n" + dashedLine);
+    }
+
+
+    public static void main(String[] args) {
+
+        sayWelcome();
+        Scanner input = new Scanner(System.in);
+
+        while (true) {
+
+            String command = input.nextLine().trim();
+
+            if (command.equals("bye")) {
+                break;
+            } else if (command.equals("list")){
+                doList();
+            } else if (command.startsWith("mark")) {
+                doMark(command);
+            } else if (command.startsWith("unmark")) {
+                doUnmark(command);
+            } else if (command.startsWith("todo")) {
+                if (counter >= 100) {
+                    System.out.println(dashedLine + "Sorry partner, the list is full!" + dashedLine);
+                    continue;
+                }
+                doToDo(command);
+            } else if (command.startsWith("deadline")) {
+                if (counter >= 100) {
+                    System.out.println(dashedLine + "Sorry partner, the list is full!" + dashedLine);
+                    continue;
+                }
+                doDeadline(command);
+            } else if (command.startsWith("event")) {
+                if (counter >= 100) {
+                    System.out.println(dashedLine + "Sorry partner, the list is full!\n" + dashedLine);
+                    continue;
+                }
+                doEvent(command);
+            } else {
+                System.out.println(dashedLine + "Something seems off partner...\n" + dashedLine);
+            }
+        }
+
+        // Closing Statement
+        System.out.println(dashedLine + "It's been a pleasure partner.\n" +
+                "Hope to see you around these parts again soon!\n" + dashedLine);
+
+    }
+}
```

Comment:
LGTM, overall good adherence to Coding standards and clear use of constants instead of using magic literals